test/test_va_api_config_attribs.cpp

```diff
@@ -28,6 +28,7 @@ namespace VAAPI {
 
 class VAAPIConfigAttribs
     : public VAAPIFixture
+    , public ::testing::WithParamInterface<std::tuple<VAProfile, VAEntrypoint> >
```

Comment:
Pedantic, but ::pair could suffice.

Reply:
tuple is nicer so you can use ::testing::get<n>

Reply:
hah, nvmind... looks like std::pair supports `get<n>` too ;)